docs/Policy/Documentation/Step_Registry_Documentation_Policy.md

```diff
@@ -9,16 +9,12 @@
     - [Process](#process)
     - [Requirements](#requirements)
 - [Automatically Generated Documentation](#automatically-generated-documentation)
-- [Markdown Resources](#markdown-resources)
-  - [Documentation](#documentation)
-  - [Editors](#editors)
-  - [Plugins](#plugins)
 
 ## Introduction
 Documenting new steps in the step registry is important because it can help use work more efficiently as well as helping users avoid duplicating work in the Step Registry. Duplicated work and lack of documentation, at the time of writing this document, seems to be an issue in the Step Registry and we would like to start solving that issue. When adding a new workflow, ref, chain, _etc._ to the Step Registry, please follow the documentation guide below to write helpful documentation for the new step.
```

Comment:
```suggestion
Documenting new steps in the step registry is important because it can help us work more efficiently as well as helping users avoid duplicating work in the Step Registry. Duplicated work and lack of documentation, at the time of writing this document, seems to be an issue in the Step Registry and we would like to start solving that issue. When adding a new workflow, ref, chain, _etc._ to the Step Registry, please follow the documentation guide below to write helpful documentation for the new step.
```